app/views/includes/two_thirds_form.html

```diff
@@ -16,10 +16,12 @@
                 <input type="hidden" name="_csrf" value="{{csrfToken}}">
                 {% block form_content %}{% endblock %}
             </form>
-            {% block save_and_close %}<a href="/sign-out">{{common.saveAndClose}}</a>{% endblock %}
+            {% block save_and_close %}<p><a href="/sign-out">{{common.saveAndClose}}</a></p>{% endblock %}
```

Comment:
Any specific reason why the `<a>` tag is now contained in a `<p>` tag?  It looks no different on screen.

Reply:
As a response to your request from 16 days ago "Can you please use CSS to add spacing above this element rather than <br> tags".
Instead of styling something, I just removed the <br> tags and wrapped the link in a paragraph.

Reply:
That's a good enough answer for me.  :)
Thank you!

Reply:
Sorry, it was David's request, not yours :)